### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by foxleigh81** [foxleigh81/node-red-contrib-todoist-api#15] [Announcements]
This release now uses the V2 API from Todoist, includes a 'get all projects' node and has some updated documentation.

<hr /><em>This discussion was created from the release <a href='https://github.com/foxleigh81/node-red-contrib-todoist-api/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by b-neufeld** [comment]
I have only just recently "discovered" Todoist and it didn't take long for me to start hunting for a way to integrate it in Node-RED. Thank you so much for sharing this project, it works like a charm so far!!
